knowledge-base/disable-logging-tracing-information-from-telerik-reporting.md

```diff
@@ -0,0 +1,102 @@
+---
+title: How To Disable Logging the Tracing Information from Telerik Reporting
+description: How to skip logging the tracing information from Telerik Reporting by your project Trace Listeners
+type: how-to
+page_title: Skip Tracing Information Logged by Telerik Reporting
```

Comment:
Either in the page_title or in the description you may the _ignore_ word for better SEO. For example:

How to ignore the tracing information from Telerik Reporting by your project Trace Listeners